CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -351,10 +352,12 @@ local function Create3DMenuBar(barAlignment, threeDPanel)
 		return totalWidth
 	end
 
-	function this:CloseAllExcept(item)
+	function this:CloseAllExceptAsync(item, ignoreModality)
 		for _, v in pairs(this:GetItems()) do
 			if v ~= item then
-				v:SetActive(false)
+				if ignoreModality or v:IsModal() then
```

Comment:
Just checking that if `ignoreModality` is false and `v:IsModal()` is also false we don't want to set that item to inactive?